seed/data_importer/migrations/0016_auto_20220119_1347.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.7 on 2022-01-19 21:47
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_importer', '0015_auto_20210712_2134'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='importfile',
+            name='export_file',
+        ),
+        migrations.RemoveField(
+            model_name='importrecord',
+            name='import_completed_at',
```

Comment:
Probably need to remove this:
https://github.com/SEED-platform/seed/blob/3029-media-volume-data/seed/data_importer/models.py#L501

Reply:
done, thanks for catching this one!